docs/BR.md

```diff
@@ -853,30 +853,26 @@ The CA SHALL update and reissue CRLs at least (i) once every twelve months and (
 No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
-
-1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
-
-In the latter case, the OCSP signing Certificate MUST contain an extension of type id-pkix-ocsp-nocheck, as
-defined by RFC6960.
+The CA SHALL provide an OCSP responder that conforms to the profile specified by RFC5019 for all Certificates issued, including the transport profile.
```

Comment:
@SamerW That's not a correct analysis. RFC 5019 is a profile of RFC 6960 - so of course 5019 is more restrictive than 6960 (or, alternatively, 6960 is more liberal). You're arguing that because it's possible to be more liberal, it's at conflict with 5019 - but not, the entire purpose of referencing 5019 is to indicate the MUST requirement nature of the nextUpdate, and then to further constrain the profile of 5019 in the Baseline Requirements.